dashboards.go

```diff
@@ -622,74 +109,3 @@ func doPushDashboard(c *cli.Context) error {
 	}
 	return nil
 }
-
-func doMigrateDashboard(c *cli.Context) error {
-	id := c.String("id")
-	if id == "" {
-		return cli.NewExitError("--id is required", 1)
-	}
-	client := mackerelclient.NewFromContext(c)
-
-	dashboard, err := client.FindDashboard(id)
-	logger.DieIf(err)
```

Comment:
Currently, because legacy support no longer supported from Dashboards API so this statement will always occur an error.